logstash-event.gemspec

```diff
@@ -25,4 +26,11 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rspec"
```

Comment:
We need to remove `spec/event.rb` from the list of files. gem build will complains if it doesn't exist.